Library/Formula/pgroonga.rb

```diff
@@ -0,0 +1,39 @@
+class Pgroonga < Formula
+  desc "PostgreSQL plugin to use Groonga as index"
+  homepage "https://pgroonga.github.io/"
+  url "http://packages.groonga.org/source/pgroonga/pgroonga-1.0.0.tar.gz"
+  sha256 "854d66bdc79e7cfceb29ae8706556a3ad1c8b32ac4535939d296966e3b19fd3e"
+
+  depends_on "pkg-config" => :build
+  depends_on "groonga"
+  depends_on "postgresql"
+
+  def install
+    # To find HOMEBREW_PREFIX/bin/pg_config
+    ENV.prepend_path "PATH", HOMEBREW_PREFIX/"bin"
```

Comment:
> This also applies to nginx. Perhaps we're better to just bundle the most popular plugins (e.g. those already in core and stuff like postgis) and move the rest to a tap.

While, `nginx` doesn't have any problem on prefix sandbox. Currently we have two choice:
* Bundle everything in one formula
* Like this PR, hacking using hard link so plugins can be as individual formulae.

So should I merge this PR?